packages/playwright-crawler/src/internals/adaptive-playwright-crawler.ts

```diff
@@ -18,7 +19,9 @@ import type { PlaywrightCrawlerOptions, PlaywrightCrawlingContext } from './play
 import { PlaywrightCrawler } from './playwright-crawler';
 import { RenderingTypePredictor, type RenderingType } from './utils/rendering-type-prediction';
 
-type Result<TResult> = { result: TResult; ok: true } | { error: unknown; ok: false };
+type Result<TResult> =
+    | { result: TResult; ok: true; logs?: LogProxyCall[] }
+    | { error: unknown; ok: false; logs?: LogProxyCall[] };
```

Comment:
May as well make `logs` required, isn't it so?

Reply:
Nope, its there only for the http run, browser run is logging right ahead (since that result is always committed)

Reply:
Ah, yeah, I made that comment too early. I would probably just add the logs attribute to the result type of the http method then, but it's a very minor detail.